site/content/docs/main/upgrade-to-1.10.md

```diff
@@ -44,19 +47,24 @@ Before upgrading, check the [Velero compatibility matrix](https://github.com/vmw
     velero install --crds-only --dry-run -o yaml | kubectl apply -f -
     ```
 
-    **NOTE:** Since velero v1.9.0 only v1 CRD will be supported during installation, therefore, the v1.9.0 will only work on kubernetes version >= v1.16
+    **NOTE:** Since velero v1.10.0 only v1 CRD will be supported during installation, therefore, the v1.10.0 will only work on kubernetes version >= v1.16
 
-1. Update the container image used by the Velero deployment and, optionally, the restic daemon set:
+1. Update the container image and objects fields used by the Velero deployment and, optionally, the Velero daemon set:
 
     ```bash
-    kubectl set image deployment/velero \
-        velero=velero/velero:v1.9.0 \
-        --namespace velero
-
-    # optional, if using the restic daemon set
-    kubectl set image daemonset/restic \
-        restic=velero/velero:v1.9.0 \
-        --namespace velero
+    # uploader_type value cloud be restic or kopia    
+    kubectl get deploy -n velero -ojson \
+    | sed "s#\"image\"\: \"velero\/velero\:v[0-9]*.[0-9]*.[0-9]\"#\"image\"\: \"velero\/velero\:v1.10.0\"#g" \
+    | sed "s#\"--default-volumes-to-restic=true\"#\"--default-volumes-to-fs-backup=true\",\"--uploader-type=$uploader_type\"#g" \
+    | sed "s#default-restic-prune-frequency#default-repo-maintain-frequency#g" \
+    | sed "s#restic-timeout#fs-backup-timeout#g" \
+    | kubectl apply -f -
+
+    # optional, if using the velero daemon set
```

Comment:
Is it intentional that `restic` was replaced by `velero`?

Reply:
I'm still keeping `restic` now, older version indeed is `restic` daemon set